bindings/pyroot_experimental/cppyy/cppyy/python/cppyy/__init__.py

```diff
@@ -55,7 +55,15 @@
 major, minor = sys.version_info[0:2]
 py_version_str = '{}_{}'.format(major, minor)
 libcppyy_mod_name = 'libcppyy' + py_version_str
-importlib.import_module(libcppyy_mod_name)
+try:
+    importlib.import_module(libcppyy_mod_name)
+except ModuleNotFoundError as err:
```

Comment:
Please check if this is `ImportError` in Python2, and if there is a way to do this in a unified manner for both Python versions.

Reply:
yes, it's ```ImportError``` in Python2
```
Python 2.7.17 (default, Apr 15 2020, 17:20:14) 
[GCC 7.5.0] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> 
>>> 
>>> import ROOT
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/massimiliano/root-devel/tries/lib/ROOT/__init__.py", line 28, in <module>
    import cppyy
  File "/home/massimiliano/root-devel/tries/lib/cppyy/__init__.py", line 58, in <module>
    importlib.import_module(libcppyy_mod_name)
  File "/usr/lib/python2.7/importlib/__init__.py", line 37, in import_module
    __import__(name)
ImportError: No module named libcppyy2_7
```

changing